### ISSUE TITLE
Reproducing repository accuracy

### ISSUE THREAD

**Issue by OmarDahroug** [anantzoid/Recurrent-Entity-Networks-pytorch#2]
Dear Anant, 

I'm have problems reproducing the accuracies you mentioned in the README file. I tried training the model on several tasks but I receive similar behaviour. The loss always reaches a loss of  around **1,7** and **stays mostly constant**. 

I train the model with the following hyperparameters:
- epochs = 200
- StepLR scheduler with:
   - step size =10
   - base_lr = 1e-2
   - base_lr = 2e-4
- batchsize = 32

I'm looking forward to your reply. 

Best,
Omar
